python/src/GaussianProcessFitter_doc.i.in

```diff
@@ -16,182 +17,74 @@ covarianceModel : :class:`~openturns.CovarianceModel`
-    = \dfrac{1}{(2\pi)^{\frac{1}{2} \sampleSize \outputDim} \left|\det\left(\mat{C}_{\vect{p}}\right)\right|^{\frac{1}{2}}}
-    \exp\left( -\dfrac{1}{2}\Tr{\left( \vect{y}-\vect{m} \right)} \mat{C}_{\vect{p}}^{-1}  \left( \vect{y}-\vect{m} \right)  \right)
+    Default value is `Basis(0)`, i.e. no trend to estimate.
 
-Let :math:`\mat{L}_{\vect{p}}` be the Cholesky factor of :math:`\mat{C}_{\vect{p}}`, i.e. the lower triangular matrix
-with positive diagonal such that :math:`\mat{L}_{\vect{p}} \,\Tr{\mat{L}_{\vect{p}}} = \mat{C}_{\vect{p}}`.
-Therefore the log-likelihood is:
+Notes
+-----
 
-.. math::
-    :label: logLikelihoodGP
+Refer to :ref:`gaussian_process_regression` (Step 1) to get all the notations and the theoretical aspects.
+We only detail here the notions related to the class.
 
-    \log \cL(\vect{\beta}, \vect{p};(\vect{x}_k, \vect{y}_k)_{1 \leq k \leq \sampleSize}) 
-    = \alpha - \log \left( \det \left(\mat{L}_{\vect{p}}\right) \right)
-    -\dfrac{1}{2}  \left\| \mat{L}_{\vect{p}}^{-1}(\vect{y}-\vect{m}_{\vect{\beta}}) \right\|^2_2
+We suppose we have a sample :math:`(\vect{x}_k, \vect{y}_k)_{1 \leq k \leq \sampleSize}`
+where :math:`\vect{y}_k = \model(\vect{x}_k)` for all :math:`k`, with :math:`\model:\Rset^\inputDim \mapsto
+\Rset^{\outputDim}` a given function.
 
-where :math:`\alpha \in \Rset` is a constant independent of :math:`\vect{\beta}` and :math:`\vect{p}`.
-The maximization of :eq:`logLikelihoodGP` leads to the following optimality condition for :math:`\vect{\beta}`:
+The class creates the Gaussian process  :math:`\vect{Y}(\omega, \vect{x})` such that the sample
+:math:`(\vect{y}_k)_{1 \leq k \leq \sampleSize}` is considered as its restriction  on
+:math:`(\vect{x}_k)_{1 \leq k \leq \sampleSize}`. It is defined by:
 
 .. math::
 
-    \vect{\beta}^*(\vect{p}^*)
-    = \argmin_{\vect{\beta}} \left\| \mat{L}_{\vect{p}^*}^{-1} \left(\vect{y} - \vect{m}_{\vect{\beta}} \right) \right\|^2_2
-
-This expression of :math:`\vect{\beta}^*` as a function of :math:`\vect{p}^*` is taken as a
-general relation between :math:`\vect{\beta}` and :math:`\vect{p}` and is substituted into
-:eq:`logLikelihoodGP`, leading to a *reduced log-likelihood* function depending solely on :math:`\vect{p}`.
-
-In the particular case where :math:`\outputDim = \dim(\vect{\sigma}) = 1` and :math:`\sigma` is a part of
-:math:`\vect{p}`, then a further reduction is possible. In this case, let :math:`\vect{q}` be
-the vector :math:`\vect{p}` in which :math:`\sigma` has been substituted by 1.
-Therefore:
-
-.. math::
+    \vect{Y}(\omega, \vect{x}) = \vect{\mu}(\vect{x}) + \vect{W}(\omega, \vect{x})
 
-    \left\| \mat{L}_{\vect{p}}^{-1}(\vect{y}-\vect{m}_{\vect{\beta}}) \right\|^2 
-    = \frac{1}{\sigma^2} \left\| \mat{L}_{\vect{q}}^{-1} \left(\vect{y} - \vect{m}_{\vect{\beta}} \right) \right\|^2_2.
+where :math:`\vect{\mu} = (\mu_1, \dots, \mu_\outputDim)` with :math:``\mu_\ell(\vect{x}) = \sum_{j=1}^{b}
```

Comment:
```suggestion
where :math:`\vect{\mu} = (\mu_1, \dots, \mu_\outputDim)` with :math:`\mu_\ell(\vect{x}) = \sum_{j=1}^{b}
```

Reply:
@jschueller : you have good eyes... thanks! :-)